reservation/test_schedule_model.py

```diff
@@ -0,0 +1,64 @@
+from reservation.models import Schedule, Appointment
+from datetime import datetime
+import pytest
+
+
+START_DAY = datetime(2023, 4, 17, 10, 0, 0)
+END_DAY = datetime(2023, 4, 17, 18, 0, 0)
+MEETING_TIME = 60
+
+
+@pytest.fixture
+def persisted_schedule(schedule):
+    schedule.professional_id.save()
+    schedule.save()
+    return schedule
+
+
+@pytest.fixture
+def persisted_schedule_pool(appointment, typeOfJob, persisted_schedule, professional, client2):
+    appointment2 = Appointment(professional_id=professional,
+                               client_id=client2,
+                               typeOfJob_id=typeOfJob,
+                               start_appointment=datetime(2023, 4, 17, 13, 0, 0),
+                               end_appointment=datetime(2023, 4, 17, 14, 0, 0),
+                               summary="")
+    appointment.typeOfJob_id.save()
+    appointment.professional_id.save()
+    appointment.client_id.save()
+    appointment.save()
+    appointment2.professional_id.save()
+    appointment2.client_id.save()
+    appointment2.typeOfJob_id.save()
+    appointment2.save()
+    return [["10:00-11:00", "11:00-12:00", "14:00-15:00", "15:00-16:00", "16:00-17:00", "17:00-18:00"],
```

Comment:
Makes sense to have these times as `DURATION`?

Reply:
I think not because I need the hours to be exactly "12:00-13:00" etc. And so it complicates the code to use it (requires loops, etc.)

Reply:
What @dkeler tries to show, that a visit can take 30 minutes, is'nt it?

Reply:
We assume that a certain professional chooses for that day the length of a meeting and all meetings will be according to the appointed time. It is possible that one professional will say that his visits on a certain day are for an hour and another professional will say that his visits on a certain day are for half an hour. Therefore in this case I checked for a professional who decided that each of his visits on the specific day chosen would take an hour

Reply:
"I choose not to die on that hill"

But refer this comment 
"And so it complicates the code to use it (requires loops, etc.)"

Could be that there is a compromise of code quality due to lack of code expertise? 

@dkeler, your decision please